docs/admins/access-override.rst

```diff
@@ -0,0 +1,84 @@
+:og:description: An access override allows someone access to (mostly electronic) RoboJackets services before they pay dues for the current semester. Typically, access overrides are provided on a case-by-case basis upon request.
+
+Access Override
+===============
+An access override allows someone access to (mostly electronic) RoboJackets services before they pay dues for the current semester.
+Typically, access overrides are provided on a case-by-case basis upon request.
+
+Self-Service
+------------
+
+.. seealso::
+   There is a :doc:`separate member-facing page for self-service overrides<../members/access-override>`.
+
+To prevent abuse, several criteria must be met before a user becomes eligible for a self-service override.
+Under normal circumstances, it should be very easy for a well-intentioned RoboJackets new member to receive a self-service override.
+
+A user's eligibility for a self-service override is visible on their details page in Nova, in the System Access panel.
+
+Self-service overrides can last between 7 and 60 days.
+The longest duration will occur if the next (earliest) future :doc:`dues package<../officers/dues/data-model>` access end date is 60 or more days in the future.
+The shortest duration will occur if the next future access end date is 7 or less days away.
+
+Self-service override criteria are divided into 2 categories: conditions and tasks.
+A condition is a system- or user-level property that cannot be rectified by the user alone (see list below). However, tasks can be completed by users.
+
+Required Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- User access must not be active
+- User must have no prior dues payments
+- User must have no active or prior access override
+- A :doc:`dues package<../officers/dues/data-model>` with the "access end" date in the future must exist
```

Comment:
Reference the actual model attribute name (mostly adding because this is done inconsistently on this page)